winit-android/src/lib.rs

```diff
@@ -62,30 +62,33 @@
 //! If your application is currently based on `NativeActivity` via the `ndk-glue` crate and building
 //! with `cargo apk`, then the minimal changes would be:
 //! 1. Remove `ndk-glue` from your `Cargo.toml`
-//! 2. Enable the `"android-native-activity"` feature for Winit: `winit = { version = "0.30.11",
+//! 2. Enable the `"android-native-activity"` feature for Winit: `winit = { version = "0.30.10",
```

Comment:
We should probably not revert this bump?

Reply:
Ooops, sorry